porting/introduction.rst

```diff
@@ -17,7 +17,7 @@ Start at :doc:`building-halium-boot` if you'd like to install the UBports Ubuntu
 .. toctree::
     :maxdepth: 1
     :name: toc-porting
-
+    halium-ubports-differences
```

Comment:
this line breaks your build https://ci.ubports.com/blue/organizations/jenkins/docs.ubports.com/detail/PR-282/2/pipeline

it's only for reading other files, not for sections in existing file